src/database-controller/config/database-controller.yaml

```diff
@@ -39,3 +39,13 @@ db-poller-max-heap-mb: 4096
 framework-watcher-max-rpc-concurrency: 150
 # Framework watcher max heap size in MB
 framework-watcher-max-heap-mb: 8192
+
+# cluster event watcher
+# Max rpc concurrency for cluster event watcher
+cluster-event-watcher-max-rpc-concurrency: 40
+# cluster event watcher max heap size in MB
+cluster-event-watcher-max-heap-mb: 2048
+# Max connection number to database in cluster event watcher.
+cluster-event-max-db-connection: 40
+# Max disk usage in internal storage for cluster event watcher
+cluster-event-watcher-max-disk-usage-percent: 80
```

Comment:
Also limit for history ? Why not move non-critical things to another DB server?

Reply:
Recorded in #4953 . We can solve this problem in the future.